examples/Jaeger.Example.WebApi.NetFx/App_Start/AutofacConfig.cs

```diff
@@ -13,6 +16,10 @@ public class AutofacConfig
 
         static AutofacConfig()
         {
+            // This is necessary to pick the correct sender, otherwise a NoopSender is used!
```

Comment:
doesn't seem related to the PR directly, is this just an oversight from the previous resolver refactoring?

Reply:
Yep, oversight from when we split up the packages. Since most people either use the NetCore package for ASP.NET (which has it‘s own example over there) or directly the configure options as in the main readme, nobody recognized.